README.adoc

```diff
@@ -12,6 +12,9 @@ middlware that you "bind" to an app, essentially providing it with an
 environment variable containing credentials (e.g. the location and
 username to use for the service).
 
+The `spring-cloud-cloudfoundry-commons` module configures the
```

Comment:
This should go in docs/src/main/asciidoc

Reply:
fixed.